common/environment/environment.hpp

```diff
@@ -15,11 +15,8 @@
     limitations under the License.
 ############################################################################## */
 
-// *** Include file generated by HIDL Version 1.3 from environment.scm ***
-// *** Not to be hand edited (changes will be lost on re-generation) ***
-
-#ifndef environment_SCM_INCL
-#define environment_SCM_INCL
+#ifndef ENVIRONMENT_INCL
```

Comment:
is this file no longer generated as the legacy comment states?

Reply:
it isn't generated, which is why I tidied these confusing comments up.
I don't know when it was generated.